chains/substrate/chain.go

```diff
@@ -35,6 +37,15 @@ func InitializeChain(cfg *core.ChainConfig) (*Chain, error) {
 	if err != nil {
 		return nil, err
 	}
+	var id msg.ChainId
```

Comment:
should this be a helper that can be tested? Otherwise lg2m

Reply:
agreed, separate function would be nice. Lets get the chain up for you and add a test.